skimage/measure/_regionprops.py

```diff
@@ -1234,6 +1481,10 @@ def _parse_docs():
                           doc, flags=re.DOTALL)
     prop_doc = {m.group(1): textwrap.dedent(m.group(2)) for m in matches}
 
+    for k, v in PROPS.items():
+        prop_doc[k] = prop_doc[v] + (
+            f"\nThis property is deprecated, use {v} instead.")
```

Comment:
```suggestion
            f"\nThis property name is deprecated, use {v} instead.")
```
(not sure? property name vs. actual property)

Reply:
I like this suggestion @mkcor @grlee77